wa/starter/pom.xml

```diff
@@ -237,12 +241,14 @@ under the License.
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-security</artifactId>
     </dependency>
-
+    <dependency>
```

Comment:
Not a big issue, but it would be nice to separate groups of dependencies by a blank line: e.g. can you add one above this?